database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -194,4 +199,6 @@ public int getItemViewType(int position) {
      * @param position  The position in the list of the view being populated
      */
     abstract protected void populateViewHolder(VH viewHolder, T model, int position);
+    
+    abstract protected void onCancelled(DatabaseError databaseError);
```

Comment:
@samtstern Thanks for the feedback! I added the JavaDoc and changed it to FirebaseListAdapter.
@puf That's a great idea! I made overriding onCancelled optional.